tests/phpunit/CRM/Contact/Import/Parser/ContactTest.php

```diff
@@ -850,6 +850,45 @@ public function testImportAmbiguousStateCountry(): void {
     $this->assertEquals(array_search('United States', $countries), $addresses['values'][1]['country_id']);
   }
 
+  /**
+   * Test importing fields with various options.
+   *
+   * Ensure we can import multiple preferred_communication_methods, single
+   * gender, and single preferred language using both labels and values.
+   *
+   * @throws \CRM_Core_Exception @throws \CiviCRM_API3_Exception
+   */
+  public function testImportFieldsWithVariousOptions() {
+    $processor = new CRM_Import_ImportProcessor();
+    $processor->setContactType('Individual');
+    $processor->setMappingFields(
+      [
+        ['name' => 'first_name'],
+        ['name' => 'last_name'],
+        ['name' => 'preferred_communication_method'],
+        ['name' => 'gender_id'],
+        ['name' => 'preferred_language'],
+      ],
+    );
```

Comment:
Console shows `PHP Parse error:  syntax error, unexpected ')' in tests/phpunit/CRM/Contact/Import/Parser/ContactTest.php on line 872`.

Reply:
Ah - Thank you! I think I added that in my last minute `civilint` efforts and didn't notice that I added a comma where it doesn't belong.

Reply:
@demeritcowboy thanks for the feedback - now this seems to be passing with that syntax fix.